### DISCUSSION TITLE
fatal error: 'cassert' file not found

### DISCUSSION THREAD

**Discussion by sooahnlee03** [facebookresearch/detectron2#5393] [Q&A]
Hello, I have a M3 MacOs and I am running into this error that says:

fatal error: 'cassert' file not found
        3 | #include <cassert>
          |          ^~~~~~~~~
    1 error generated.
    error: command '/usr/bin/clang' failed with exit code 1
    [end of output]

note: This error originates from a subprocess, and is likely not a problem with pip.

This error came up after running this code: python -m pip install -e detectron2, after doing the git clone command.
Does anyone know how to handle this error?

**Comment by 09001413** [comment]
same issue here
